src/templates/partials/layout.html.hbs

```diff
@@ -39,7 +39,7 @@
             <tbody>
               <tr>
                 <td>
-                  <img alt="Snapshot logo" class="logo-light" src="{{host}}/images/logo.png" />
```

Comment:
do we need `logo.png`?

Reply:
Kept there in case we need to generate another variant